### PR TITLE
Get started

### PR DESCRIPTION
**Pull request by Enhmunh-E**
As this is a Storybook project there is no need to start the project by yarn start. So I think newcomers won't understand how to start the project.
Added a readme.md

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Storybook for TSParticles
```
Comment (matteobruni) [inline docs/README.md:1]: tsParticles instead of TSParticles

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Easily create highly customizable particles, confetti and fireworks animations and use them as animated backgrounds for your website.
+
+Contact: matteobruni
+
+## How To Run
+
+1. `yarn install` at project root level
```
Comment (matteobruni) [inline docs/README.md:9]: `yarn` is enough

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Storybook for TSParticles
+
+Easily create highly customizable particles, confetti and fireworks animations and use them as animated backgrounds for your website.
+
+Contact: matteobruni
```
Comment (matteobruni) [inline docs/README.md:5]: Instead of contact information, use the main website: https://particles.js.org

**Review by matteobruni — CHANGES_REQUESTED** [review]

**Review by matteobruni — APPROVED** [review]

**Comment by Enhmunh-E** [comment]
can you add hacktoberfest label, please? @matteobruni

**Pull request merged**